pkg/test/resourcefixture/testdata/basic/containerattached/v1beta1/containerattachedclusterbasic/_http.log

```diff
@@ -316,24 +140,36 @@ X-Xss-Protection: 0
     "@type": "type.googleapis.com/google.cloud.gkemulticloud.v1.OperationMetadata",
     "createTime": "2024-04-01T12:34:56.123456Z",
     "endTime": "2024-04-01T12:34:56.123456Z",
-    "target": "projects/${projectId}/locations/us-west1/attachedClusters/kcc-attached-cluster-127",
+    "target": "projects/${projectNumber}/locations/us-west1/attachedClusters/kcc-attached-cluster-127",
     "verb": "create"
   },
-  "name": "projects/${projectId}/locations/us-west1/operations/${operationID}",
+  "name": "projects/${projectNumber}/locations/us-west1/operations/${operationID}",
   "response": {
     "@type": "type.googleapis.com/google.cloud.gkemulticloud.v1.AttachedCluster",
+    "binaryAuthorization": {
+      "evaluationMode": "DISABLED"
+    },
+    "createTime": "2024-04-01T12:34:56.123456Z",
     "description": "Test attached cluster",
     "distribution": "eks",
+    "etag": "abcdef0123A=",
     "fleet": {
+      "membership": "projects/${projectNumber}/locations/global/memberships/kcc-attached-cluster-127",
       "project": "projects/${projectNumber}"
     },
+    "kubernetesVersion": "1.28",
     "loggingConfig": {},
-    "name": "projects/${projectId}/locations/us-west1/attachedClusters/kcc-attached-cluster-127",
+    "monitoringConfig": {
+      "managedPrometheusConfig": {}
+    },
+    "name": "projects/${projectNumber}/locations/us-west1/attachedClusters/kcc-attached-cluster-127",
     "oidcConfig": {
       "issuerUrl": "https://oidc.eks.us-west-2.amazonaws.com/id/A115FE1C770C2452C75219524036FC0F"
     },
     "platformVersion": "1.27.0-gke.2",
-    "state": "RUNNING"
+    "state": "RUNNING",
+    "uid": "111111111111111111111",
+    "updateTime": "2024-04-01T12:34:56.123456Z"
```

Comment:
Could you update the MockGCP to match the response metadata? This is important for KCC to monitor the GCP service behavior and maintain the available features (etag, updateTime, etc).